src/unitxt/processors.py

```diff
@@ -237,4 +237,6 @@ def process_value(self, text: Any) -> Any:
 
 class LiteralEval(FieldOperator):
     def process_value(self, text: Any) -> Any:
+        if not text:
```

Comment:
Can you make this more explicit? (`if text is None or text == "":`)
Also any chance you can raise `ValueError` with detailed message in case that `not is instance(text, str)`
And thanks againg for the contributions @antonpibm !